projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-comments-endpoint.php

```diff
@@ -149,57 +211,58 @@ function callback( $path = '', $blog_id = 0, $object_id = 0 ) {
 		if ( $args['number'] < 1 ) {
 			$args['number'] = 20;
 		} elseif ( 100 < $args['number'] ) {
-			return new WP_Error( 'invalid_number',  'The NUMBER parameter must be less than or equal to 100.', 400 );
+			return new WP_Error( 'invalid_number', 'The NUMBER parameter must be less than or equal to 100.', 400 );
 		}
 
 		if ( false !== strpos( $path, '/posts/' ) ) {
-			// We're looking for comments of a particular post
-			$post_id = $object_id;
+			// We're looking for comments of a particular post.
+			$post_id    = $object_id;
 			$comment_id = 0;
 		} else {
-			// We're looking for comments for the whole blog, or replies to a single comment
+			// We're looking for comments for the whole blog, or replies to a single comment.
 			$comment_id = $object_id;
-			$post_id = 0;
+			$post_id    = 0;
 		}
 
-		// We can't efficiently get the number of replies to a single comment
+		// We can't efficiently get the number of replies to a single comment.
 		$count = false;
 		$found = -1;
 
-		if ( !$comment_id ) {
-			// We can get comment counts for the whole site or for a single post, but only for certain queries
-			if ( 'any' === $args['type'] && !isset( $args['after'] ) && !isset( $args['before'] ) ) {
+		if ( ! $comment_id ) {
+			// We can get comment counts for the whole site or for a single post, but only for certain queries.
+			if ( 'any' === $args['type'] && ! isset( $args['after'] ) && ! isset( $args['before'] ) ) {
 				$count = $this->api->wp_count_comments( $post_id );
 			}
 		}
 
 		switch ( $args['status'] ) {
-		case 'approved' :
-			$status = 'approve';
-			if ( $count ) {
-				$found = $count->approved;
-			}
-			break;
-		default :
-			if ( ! current_user_can( 'edit_posts' ) ) {
-				return new WP_Error( 'unauthorized', 'User cannot read non-approved comments', 403 );
-			}
-			if ( 'unapproved' === $args['status'] ) {
-				$status = 'hold';
-				$count_status = 'moderated';
-			} elseif ( 'all' === $args['status'] ) {
-				$status = 'all';
-				$count_status = 'total_comments';
-			} else {
-				$status = $count_status = $args['status'];
-			}
-			if ( $count ) {
-				$found = $count->$count_status;
-			}
+			case 'approved':
+				$status = 'approve';
+				if ( $count ) {
+					$found = $count->approved;
+				}
+				break;
+			default:
+				if ( ! current_user_can( 'edit_posts' ) ) {
+					return new WP_Error( 'unauthorized', 'User cannot read non-approved comments', 403 );
+				}
+				if ( 'unapproved' === $args['status'] ) {
+					$status       = 'hold';
+					$count_status = 'moderated';
+				} elseif ( 'all' === $args['status'] ) {
+					$status       = 'all';
+					$count_status = 'total_comments';
+				} else {
+					$status = $count_status = $args['status']; // phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.Found
```

Comment:
Why not breaking it down here?

Reply:
Fear of change, mostly: 78869eefea1a077b78fe102bfd854937179df0c0